### ISSUE TITLE
\t weirdness in BBEdit

### ISSUE THREAD

**Issue by otter451** [adobe-fonts/source-code-pro#266]
I am having problems with Source Code Pro in BBEdit.
 
It may be a BBEdit thing but I only experience the problem in BBEdit with SCP.
 
The issue is that \t gets replaced with what looks like a capital ‘A' - if I set bbedit to display invisibles, I get the usual capital delta ‘𝝙' wherever there is a \t.
 
Normal behaviour (with Menlo for example)
![image001](https://user-images.githubusercontent.com/80163951/110214225-c14cf480-7ef7-11eb-8b9e-3372f2e38a5e.png)

Menlo (with show invisibles set)
![image002](https://user-images.githubusercontent.com/80163951/110214244-d6c21e80-7ef7-11eb-827e-4360a8e2c945.png)

Source Code Pro
![image003](https://user-images.githubusercontent.com/80163951/110214264-eb061b80-7ef7-11eb-8a16-91d75399ff6f.png)

Source Code Pro (show invisibles)
![image004](https://user-images.githubusercontent.com/80163951/110214289-08d38080-7ef8-11eb-826f-1bb922ff2cfd.png)

**Comment by punchcutter** [comment]
This doesn't happen for me. Which exact font is this with? OTF or TTF? Which weight?

**Comment by otter451** [comment]
I am using TTF and the issue effects all weights/variants

Running latest versions of Catalina + SCP + BBEdit

I **don't** get the issue on my production machine which runs latest Mojave + BBEdit and SCP v1.1017

EDIT: I just removed SCP from the Catalina machine, restarted and installed the ancient SCP v1.017 and the issue goes away.  Rinse and repeat with SCP v2.030 and the issue is back (all variants).  I tried it a few more times just to be sure it is repeatable.  Something about \t has changed in SCP that BBEdit doesn't like... 

I am loath to mess around with my production machine, but will do so if that would be helpful.

Is it possibly the same issue as https://github.com/microsoft/cascadia-code/issues/183 ?

**Comment by punchcutter** [comment]
Yes, that's exactly the same issue. Most fonts don't actually contain the encoded tab character, but for some reason the old Apple ATSUI seems to want U+000D (which has nothing to do with the tab character) which was removed from Source Code Pro almost 5 years ago. I think we can probably put it back in without any issues and that should solve the issue for you. Here's a quick test of adding that character back if you want to try using these TTFs. 
[TTF.zip](https://github.com/adobe-fonts/source-code-pro/files/6096715/TTF.zip)

**Comment by otter451** [comment]
That fixes it!

Thanks for your help & time @punchcutter
